tests/test_enterprise/api/test_views.py

```diff
@@ -1482,7 +1482,7 @@ def create_course_enrollments_context(
             [{
                 'course_mode': 'audit',
                 'course_run_id': 'course-v1:edX+DemoX+Demo_Course',
-                'lms_user_id': 1,
+                'lms_user_id': 10,
```

Comment:
`1` ID is being used for 1st user record that we are creating in setup file in the test class.